gc-metrics.js

```diff
@@ -17,14 +17,16 @@ const gcTypes = {
 
 const noop = () => {};
 
-export default () => {
+export default client => {
+  const promClient = client || { Counter };
```

Comment:
Nice ! I'm working on it, but i'm a little busy, in period of University tests :/